The function get_mci_clk_rate is used inside atmel_mci driver
but never defined. Fix this typo.

Signed-off-by: Claudio Mignanti <c.mignanti@gmail.com>
---
 drivers/mmc/atmel_mci.c |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/drivers/mmc/atmel_mci.c b/drivers/mmc/atmel_mci.c
index 3946ffe..628aac9 100644
--- a/drivers/mmc/atmel_mci.c
+++ b/drivers/mmc/atmel_mci.c
@@ -67,7 +67,7 @@ static void mci_set_mode(unsigned long hz, unsigned long blklen)
 	unsigned long bus_hz;
 	unsigned long clkdiv;
 
-	bus_hz = get_mci_clk_rate();
+	bus_hz = get_mck_clk_rate();
 	clkdiv = (bus_hz / hz) / 2 - 1;
 
 	pr_debug("mmc: setting clock %lu Hz, block size %lu\n",
-- 
1.7.0.4

